### PR TITLE
183730545 - update seeds with pythnet

### PR DESCRIPTION
**Pull request by kamtarnowski**
#### What's this PR do?
PR removes useless network variable placed in seeds.

#### How should this be manually tested?
Run `rails db:seed` and check for errors.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183730545)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: script/add_current_epoch.rb [hunk 1]
```diff
@@ -45,6 +45,8 @@ def solana_rpc_client(network)
     next
   end
   
+  next unless block_time
```
Comment (maciejkocylapc) [inline script/add_current_epoch.rb:48]: this should be in a different PR

**Review by maciejkocylapc — CHANGES_REQUESTED** [review]

File: script/add_current_epoch.rb (hunk 1 above)
Comment (kamtarnowski) [reply script/add_current_epoch.rb:48]: Ok, placing it in https://www.pivotaltracker.com/story/show/183862308

**Review by maciejkocylapc — APPROVED** [review]

**Review by kbiala — APPROVED** [review]
